documentation/modules/auxiliary/client/hwbridge/connect.md

```diff
@@ -0,0 +1,92 @@
+## Overview
+
+This module connects to any Hardware device that supports the HWBridge API.  For details
+on the HWBridge API see [API Reference].  On successful connection to a HW Bridge a
+hwbridge session will be established.
+
```

Comment:
I'm not sure where the best place for the following documentation would go, but my guess is here.
I know this is a 'generic' hwbridge connector, but I think its important to document some of the things that are in this PR.

1. an amazon/ebay type link where someone could buy a compatible adapter. Or maybe list brands of ones that have been confirmed to work.  Similar to how we list software versions that have been verified.  This will make it easier for interested parties to get their hands on known good devices.  I've shared the amazon link and info for my device in one of my comments in the PR itself. 
2. Similar to above, I think there are any specific instructions for using the USB/BLUETOOTH ones, that should also be included.  I assume USB is just plug it in, but if any `apt-get` packages or drivers are needed, good place to put that.  For BT, I wrote up exactly what I had to do to get mine working (for my own repeatability), may be good to include that as well so someone doesn't spend 60min figuring it out like i did.

If @zombieCraig thinks this is a good idea and place to put this knowledge, maybe @wvu can finally have his opportunity to teach me how to submit a PR to a PR.  I don't mind adding to the docs!